stl/inc/bitset

```diff
@@ -68,6 +68,7 @@ public:
         size_t _Mypos; // position of element in bitset
     };
 
+private:
     static _CONSTEXPR23 void _Validate(const size_t _Pos) noexcept { // verify that _Pos is within bounds

```

Comment:
After current changes, it would be better to make `_Validate` always `constexpr`.
```suggestion
    static constexpr void _Validate(const size_t _Pos) noexcept { // verify that _Pos is within bounds
```

Reply:
Yes, thanks.
1. Do you think we should also enhance this check in constexpr contexts? In release mode(when `_ITERATOR_DEBUG_LEVEL==0`), we will bypass this check in cases where `_Pos>_Bit` but `_Pos/64<=_Word`). (I find it not easy to perfectly implement this check, as `is_constant_evaluated` is not available in early versions).
2. I find it quite annoying to check `_CONSTEXPR??`-ness for trivial internal methods, as it is dependent on all caller methods.  think this is unnecessary burden, what do you think about it?

Reply:
> (I find it not easy to perfectly implement this check, as `is_constant_evaluated` is not available in early versions)

This is doable. If we need to do different things between constant and non-constant evaluations in a pre-C++20 constexpr function, we can use `_Is_constant_evaluated`. (Other standard library implementations also have their internal versions of `is_constant_evaluated`.)